apps/betterangels-backend/clients/schema.py

```diff
@@ -153,14 +153,14 @@ class Mutation:
     def create_client_document(self, info: Info, data: CreateClientDocumentInput) -> ClientDocumentType:
         with transaction.atomic():
             user = cast(User, get_current_user(info))
+            permission_group = get_user_permission_group(user)
```

Comment:
this is just a lil refactor. I moved permission checks to the top of any op they were in